README.md

```diff
@@ -1,2 +1,69 @@
-# opseng-challenge
-Wave OpsEng Development Challenge
+# Wave OpsEng Development Challenge
+
+Applicants for the [Operations Engineer career](https://www.waveapps.com/about-us/jobs/operations-engineer/) at Wave must complete the following challenge, and submit a solution prior to the interviewing process. This will help the interviewers assess your strengths, and frame the conversation through the interview process. Take as much time as you need, however we ask that you not spend more than a few hours.
```

Comment:
> prior to the interviewing process.

@nduthoit Is this the case for us? I believe previously when we discussed this, we talked about doing a phone screen first, and then asking them to complete the code challenge.